index.html

```diff
@@ -68,7 +80,7 @@
 
             Promise.all([imageryPromise, elevationPromise]).then(() => itowns.viewer.init());
 
-            menuGlobe.addGUI('RealisticLighting',false,function(newValue) {itowns.viewer.setRealisticLightingOn(newValue);});
+            menuGlobe.addGUI('RealisticLighting',false,(newValue) => {itowns.viewer.setRealisticLightingOn(newValue);});
 
            	itowns.viewer.addEventListener('globe-loaded', function() {
             	console.log("Globe Loaded");
```

Comment:
Actually, we could have several example html, one basic like it is now, another with all the controls etc... WDYT?